l10n_ch_qr_bill_scan/wizard/account_invoice_import.py

```diff
@@ -119,6 +119,16 @@ def _read_swiss_qr_code(self, bill_img):
 
         if not locations:
             return False
+
+        try:
+            # IndexError: index 0 is out of bounds for axis 0 with size 0
+            # is raised if size of the array is 0
+            if (locations[0].size == 0) or (locations[1].size == 0):
+                return False
+        except IndexError:
+            # IndexError: list index out of range
+            return False
```

Comment:
```suggestion
            # IndexError: list index out of range
            raise UserError(_("QR is not recognized correctly try to regenerate invoice"))
```

Reply:
This is additional UserError, it's not needed. I would say we can combine this exception with this condition:
https://github.com/yvaucher/l10n-switzerland/blob/27bdcf2ea39815ef93bc208d079817039c033640/l10n_ch_qr_bill_scan/wizard/account_invoice_import.py#L120
as it's the same, but I can't guess how)

Reply:
if you return False what will see user in UI?

Reply:
Same error for all False returned-> https://github.com/OCA/edi/blob/7b4a97dce95a39fe07061f60103eee6982356f93/account_invoice_import/wizard/account_invoice_import.py#L86

Reply:
I checked this. Each time False returned pretty message appears)